.gitignore

```diff
@@ -187,4 +187,6 @@ js
 .DS_Store
 JuPedSim-blx.bib
 _tex/
-titlepage.tex
\ No newline at end of file
+titlepage.tex
+
+/scripts/deps/
```

Comment:
Please remove this ignore. It is user specific where you create the dependencies folder.